tests/test_federal_senate_dataset.py

```diff
@@ -1,41 +1,92 @@
-import os
 from tempfile import gettempdir
-from unittest import main, skipIf, TestCase, TestLoader
+from unittest import TestCase
+from unittest.mock import patch
+import pandas as pd
+import os
```

Comment:
[Isort](https://pypi.python.org/pypi/isort) is awesome. I added a plugin on my Atom.